packages/schematics/angular/application/index.ts

```diff
@@ -94,6 +94,7 @@ export default function (options: ApplicationOptions): Rule {
       mergeWith(
         apply(url('./files'), [
           options.minimal ? filter(minimalPathFilter) : noop(),
+          options.skipGit ? filter(path => !path.endsWith('gitignore')) : noop(),
```

Comment:
Could you check the whole path?  `!path.endsWith('/__dot__gitignore')`?

Reply:
fixed this (plus linting issue in generated schema file)